src/components/BallotPage.tsx

```diff
@@ -64,6 +65,8 @@ const BallotContent: React.FC<{ ballotId: number; onOpenSidebar?: () => void }>
 
   const electionType = meta?.type;
   const compatibleScopes = electionType ? electionTypeCompatibleScopes(electionType) : undefined;
+  const showCandidatesTab =
```

Comment:
can we make this available to take into consideration the fact that it is a live election?

Reply:
fixed 👍